src/reducers/accounts.js

```diff
@@ -96,7 +96,7 @@ export function serializeAccounts(accounts: any): Account[] {
   return accounts ? accounts.map(accountModel.decode) : []
 }
 
-export function deserializeAccounts(accounts: Account[]) {
+export function deserializeAccounts(accounts: Account[]): Array<any> {
```

Comment:
I think now we don't send<->receive Accounts types anymore (we always encode them to AccountRaw) so we should be able to keep the type and remove the horrible thing here:

⏬

Reply:
yeah that'(s fixed in another PR :D